code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -120,8 +120,8 @@
 	air.merge(parent_pipeline.air)
 	for(var/obj/machinery/atmospherics/components/reference_component in parent_pipeline.other_atmosmch)
 		reference_component.replacePipenet(parent_pipeline, src)
-	other_atmosmch.Add(parent_pipeline.other_atmosmch)
-	other_airs.Add(parent_pipeline.other_airs)
+	other_atmosmch |= parent_pipeline.other_atmosmch
+	other_airs |= parent_pipeline.other_airs
```

Comment:
I don't think? the other_airs portion of this is needed, but let's play it safe shall we?

Reply:
wouldn't this play a role when we combine two pipelines with machineries of their own and therefore gasmixes?

Reply:
Unless it's possible to access the same internal gasmix from two sides of a component, no. Good to plan for tho.